lib/tokenVersions.js

```diff
@@ -111,12 +111,24 @@ export async function ensureTokenVersion({tokenizerId}) {
   return tokenVersion;
 }
 
+/**
+ * Gets a token version by either the version id or the tokenizerId.
+ *
+ * @param {object} options - Options to use.
+ * @param {number} [options.id] - An optional version id.
+ * @param {string} [options.tokenizerId] - An optional tokenizerId.
+ *
+ * @throws {Error} - If neither an id or tokenizerId is provided.
+ *
+ * @returns {object} A tokenizerVersion object.
+ */
 export async function get({id, tokenizerId}) {
   const query = {};
-  if(!(id && tokenizerId)) {
+  if(!((typeof id === 'number') || (typeof tokenizerId === 'string'))) {
     throw new TypeError('Either "id" or "tokenizerId" must be given.');
   }
-  if(id) {
+  // if the version id is 0 we need it in the query
+  if(id >= 0) {
     query['tokenVersion.id'] = id;
   }
   if(tokenizerId) {
```

Comment:
I can't get github to let me make a proper suggestion here, but since the string type check was removed above with another suggestion, we need this here:

```js
  if(tokenizerId !== undefined) {
    if(typeof tokenizerId !== 'string') {
      throw new TypeError('"tokenizerId" must be a string.');
    }
    query['tokenVersion.tokenizerId'] = tokenizerId;
  }
```

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-tokenization/pull/5/commits/30819aef7a881309ca7ecc2a2a7732467fe81538